doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -1655,37 +1655,37 @@ connect <host> <[+]port>
 
   Module: core
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-listen <port> <type> [options] [flag]
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+listen [ip] <port> <type> [options [flag]]
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  Description: opens a listening port to accept incoming telnets; type must be one of "bots", "all", "users", "script", or "off". Prefixing the port with a plus sign will make eggdrop accept SSL connections on it.
+  Description: opens a listening port to accept incoming telnets; type must be one of "bots", "all", "users", "script", or "off". Prefixing the port with a plus sign will make eggdrop accept SSL connections on it. An IP may optionally be listed before the mandatory port argument. If no IP is specified, 0.0.0.0 (all available addresses) is used.
```

Comment:
the default is "listen-addr" (deprecated), or, if that is not set "all interfaces" (0.0.0.0 suggests IPv4 only which isn't necessarily true)